### PR TITLE
Updated options page styles

### LINKED ISSUE #40: Integrate CC-Vocabulary in text element in options page
CC-vocabulary is integrated into other elements like checkboxes, tabs, buttons. We need to add it to text elements like paragraphs and headings too.

**How to do it**
Note - [CC-vocabulary](https://opensource.creativecommons.org/cc-vocabulary/styleguide/) is already linked into the project, therefore you just need to add the appropriate classes(which you can see after inspecting the element in dev tools) and mimic the markup(if different) to change the styles.

In this issue, the following elements must be targetted:
- The heading and the Note in the _Filters_ tab.
- The Note in the _Bookmarks_ tab.
- Heading in the _Dark-Mode_ tab.
- Heading and paragraph in _About_ tab.

Make sure to change the heading tags to adjust the size of headings.

## Reference
- [Paragraph section of CC-vocab styleguide](https://opensource.creativecommons.org/cc-vocabulary/styleguide/#/Elements/Paragraph)
- [Heading section of CC-vocab styleguide](https://opensource.creativecommons.org/cc-vocabulary/styleguide/#/Elements/Heading)

### PR DESCRIPTION
**Pull request by sharmaadityaHQ**
Fixes #40 

**Description**
Updated with CC styles in options page

**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

**Comment by sharmaaditya570191** [comment]
Screenshot from 2019-12-13 20-31-37
Screenshot from 2019-12-13 20-31-48
Screenshot from 2019-12-13 20-31-53

**Comment by sharmaaditya570191** [comment]
Please have a look @makkoncept and suggest changes if any. Thanks.

File: src/options/options.html [hunk 1]
```diff
@@ -24,21 +24,22 @@
...
-            </ul>
+            <p class="vocab paragraph inherit-colored"><strong>Note:</strong>
+              <ul class="margin-bottom-medium">
+                <li>Changes will take effect after you click the save button at the bottom.</li>
+              </ul>
+            </p>
           </div>
 
-          <h4>I want something that (use-case)</h4>
+          <h6 class="vocab heading inherit-colored">I want something that (use-case)</h6>
```
Comment (makkoncept) [inline src/options/options.html]: You can remove `inherit-colored` class heading and paragraph

**Review by makkoncept — CHANGES_REQUESTED** [review]

File: src/options/options.html (hunk 1 above)
Comment (sharmaaditya570191) [reply src/options/options.html]: I have made the changes now.

**Comment by makkoncept** [comment]
Thanks for working on this

**Pull request merged**